contrib/helm/harbor/README.md

```diff
@@ -30,26 +81,29 @@ The command deploys Harbor on the Kubernetes cluster in the default configuratio
 To uninstall/delete the `my-release` deployment:
 
 ```bash
-$ helm delete my-release
+helm delete my-release
 ```
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
 ## Configuration
 
-The following tables lists the configurable parameters of the Percona chart and their default values.
+The following tables lists the configurable parameters of the Harbor chart and the default values.
 
 | Parameter                  | Description                        | Default                 |
 | -----------------------    | ---------------------------------- | ----------------------- |
 | **Harbor** |
-| `externalDomain`               | domain harbor will run on (https://*harbor.url*/) |`harbor.192.168.99.100.xip.io`   |
-| `tls_crt`               | TLS certificate to use for Harbor's https endpoint | see values.yaml |
-| `tls_key`               | TLS key to use for Harbor's https endpoint | see values.yaml |
-| `ca_crt`               | CA Cert for self signed TLS cert | see values.yaml |
+| `harborImageTag`     | The tag for Harbor docker images | `v1.4.0` |
+| `externalDomain`       | Harbor will run on (https://*externalDomain*/). Make sure this FQDN resolves to the K8s Ingress Controller IP. | `harbor.my.domain` |
+| `insecureRegistry`     | Set to true if setting Harbor Registry as insecure-registries for docker | `false` |
```

Comment:
Let's explicitly say if `insecureRegistry` is set to true, the tls cert of ingress controller will not be set by helm.